backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/services/EMFValidationService.java

```diff
@@ -39,6 +40,7 @@
  * @author gcoutable
  */
 @Service
+@ConditionalOnMissingBean
```

Comment:
Why do you need that to be conditional? If you are using EMF, why wouldn't you use the EMF validation service?

Reply:
I'm not using EMF. The addition of these annotations make it possible for my application not to depend on EMF, that's the point :-)

Reply:
I removed these useless @ConditionalOnMissingBean.